src/views/Home.jsx

```diff
@@ -1,11 +1,62 @@
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import Button from '../components/Button';
 import './Home.css';
+import { useState } from 'react';
+// import { createCollection } from '../api/firebase';
+
+export function Home({ setListToken }) {
+	const [retrievedToken, setRetrievedToken] = useState('');
+
+	/**
+	 * This function generates a new unique Token corresponding to a new shopping list.
+	 */
+	const createNewList = () => {
+		const newToken = generateToken();
+		setRetrievedToken(newToken);
```

Comment:
One small suggestion is to use setListToken(newToken) here on line 15 instead of setRetrievedToken, this will redirect users immediately to the list page